go/tools/builders/compile.go

```diff
@@ -49,6 +49,7 @@ func run(args []string) error {
 	if err := goenv.update(); err != nil {
 		return err
 	}
+	absOutput := abs(*output)
```

Comment:
Without any comment here, we'll probably forget why this is here and remove it in six months. :)

It would also be good to have a more detailed comment on `abs` itself that references `path_windows.go` as you did in the commit message.

Reply:
Done.